portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Susana's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Welcome to My Portfolio:)!</h1>
+      <p>My name is Susana Perez<p> 
+      <h2>About me</h2>
+      <img src="images/download.png" alt="Front facing picture of a woman" <div id="size"></div>
+      <p> I am a CS student at the University of Texas at El Paso. I just finished my first year and
+          I am a Google STEP intern this summer!
+          I am always trying to improve my technical abilities and I love meeting and working with 
+          new people because I always get to learn something different.</p>
+      <h2> Cool projects I've worked on</h2>
+       <p>I like to attend Hackathons since I get to gain experience on the field, create cool projects, and work with different people.
+          My favorite project I've worked on was the prototype of an app that teaches financial education to college students. 
+          The app has a pet companion that keeps track of the user's expenses and analyzes them to offer financial tips
+          and ways to reduce monthly expenses.</p>
+      <h2> Pictures!</h2>
+      <img src="images/GoogleOffice.jpg" alt="Picture of a building" <div id="size"></div>
+      <p> Last January I visited Google's Playa Vista Office in California.</p>
+      <img src="images/marinaDelRey.jpg" alt="Person on a pier" <div id="size"></div>
+      <p> I was there for a just a couple of days but I had a lot of fun, this is me visiting Marina del Rey's pier.</p>
+      <img src="images/beach.jpg" alt="Sunset at sea" <div id="size"></div>
+      <p> I took this picture on my last day in California, I really enjoyed seeing the waves and the orange color of the sky.</p>
+      <h2> Random Facts about me </h2> 
+      <p>Click this button to see a random fact about me:</p>
+      <button onclick="addRandomFact()">Click Here!</button>
+      <div id="fact-container"></div>
+      <p>
+      <a href="https://www.linkedin.com/in/susana-perez-juarez/">LinkedIn profile</a>
+      <p> </p>
```

Comment:
What is this for? if you want to create an empty space you need to use a div with styling